async_fuse/src/byte_slice.rs

```diff
@@ -77,11 +87,12 @@ impl<'a> ByteSlice<'a> {
         Ok(ret)
     }
 
+    /// Fetch remaining bytes and build a slice of target instances
     #[allow(dead_code)]
-    pub(crate) fn fetch_all_as_slice<T: FuseAbiData>(&mut self) -> anyhow::Result<&'a [T]> {
+    pub fn fetch_all_as_slice<T: FuseAbiData>(&mut self) -> anyhow::Result<&'a [T]> {
         let elem_len: usize = mem::size_of::<T>();
 
-        if self.data.len() % elem_len != 0 {
+        if self.data.len().overflow_rem(elem_len) != 0 {
```

Comment:
The same with line 22.
Consider using `wrapping_rem`.

Reply:
same above